final_task/README.md

```diff
@@ -23,3 +24,36 @@ Installation recommendation rss-reader:
 6)To run the utility, type in the terminal "rss-reader" then a space and url on news
 Example : rss-reader  https://news.yahoo.com/rss
 
+
+To use caching you must have a postgresql database on your computer or laptop
+with default parameters:
+    database="postgres",
+    user="postgres",
+    password="1",
+    host="127.0.0.1",
+    port="5432"    
+If you do not follow these commands:    
+1)Open terminal
+2)sudo apt-get install postgresql
+3)sudo -u postgres psql
+4)\password
+5)1
+6)1
```

Comment:
Did exactly as written here, does not work for me:
![k](https://user-images.githubusercontent.com/43743521/69369643-7761e000-0cad-11ea-8fe7-d02132375f18.png)

Reply:
changed README.md.Please, try again